docs/guides/fractional-gates.ipynb

```diff
@@ -0,0 +1,354 @@
+      "text/plain": [
+       "<Figure size 1625.27x451.5 with 1 Axes>"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit import QuantumCircuit\n",
+    "from qiskit.circuit import Parameter\n",
+    "from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager\n",
+    "from qiskit.visualization.timeline import draw as draw_timeline, IQXSimple\n",
+    "\n",
+    "from qiskit_ibm_runtime import QiskitRuntimeService\n",
+    "\n",
+    "\n",
+    "num_qubits = 5\n",
+    "num_time_steps = 3\n",
+    "rx_angle = 0.1\n",
+    "rzz_angle = 0.1\n",
+    "\n",
+    "ising_circuit = QuantumCircuit(num_qubits)\n",
+    "for i in range(num_time_steps):\n",
+    "    # rx layer\n",
+    "    for q in range(num_qubits):\n",
+    "        ising_circuit.rx(rx_angle, q)\n",
+    "    for q in range(1, num_qubits-1, 2):\n",
+    "        ising_circuit.rzz(rzz_angle, q, q+1)\n",
+    "    # 2nd rzz layer\n",
+    "    for q in range(0, num_qubits-1, 2):\n",
+    "        ising_circuit.rzz(rzz_angle, q, q+1)\n",
+    "    ising_circuit.barrier()\n",
+    "ising_circuit.draw('mpl')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "983bbe11-fa20-4043-a146-66af18cae054",
+   "metadata": {},
+   "source": [
+    "Then specify two backend objects, which with fractional gates enabled, another which has them disabled and transpile them both."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "09ef3ed0-7126-45a2-a876-35a4f540c949",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "service = QiskitRuntimeService()\n",
+    "backend_fractional = service.backend('ibm_torino', use_fractional_gates=True)\n",
+    "backend_conventional = service.backend('ibm_torino', use_fractional_gates=False)\n",
+    "\n",
+    "pm_fractional = generate_preset_pass_manager(optimization_level=3, backend=backend_fractional, scheduling_method=\"alap\")\n",
+    "pm_conventional = generate_preset_pass_manager(optimization_level=3, backend=backend_conventional, scheduling_method=\"alap\")\n",
+    "\n",
+    "ising_circuit_fractional = pm_fractional.run(ising_circuit)\n",
+    "ising_circuit_conventional = pm_conventional.run(ising_circuit)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cf108858-b8ee-44a8-b252-3e7748b691f8",
+   "metadata": {},
+   "source": [
+    "Then display the timeline of the circuit using the two types of gates"
```

Comment:
```suggestion
    "Display the timeline of the circuit using the two types of gates"
```
The previous sentence already started with "Then", so it's nicer to omit it here.